support/drop-installer.sh

```diff
@@ -32,19 +32,34 @@ main() {
     need_cmd uname
     need_cmd rm
     need_cmd tar
+    need_cmd unzip
```

Comment:
How about putting this into the `windows` case in the extension detection block? As it stands, if someone happens to have a Unixy system without `unzip`, the installer will error out even though it could work. I have to admit that I'm pretty sure that this scenario may never come up in practice, but if I'm not missing something, it should be a pretty trivial adjustment to make.

(While we're at it, I suppose the `need_cmd tar` could be moved to the default case in the `case`.)